src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -2119,8 +2119,17 @@ public sealed partial class String : System.Collections.Generic.IEnumerable<char
         public System.String ToUpper(System.Globalization.CultureInfo culture) { throw null; }
         public string ToUpperInvariant() { throw null; }
         public string Trim() { throw null; }
+#if netcoreapp11
```

Comment:
I think we do'nt need this anymore. @weshaggard ? (I can't find that PR where you explained this alreadY)